_rules/letter-spacing-not-important-24afc2.md

```diff
@@ -0,0 +1,261 @@
+---
+id: 24afc2
+name: Letter spacing in `style` attributes is not `!important`
+rule_type: atomic
+description: |
+  This rule checks that the `style` attribute is not used to prevent adjusting `letter-spacing` by using `!important`, except if it's at least 0.12 times the font size.
+accessibility_requirements:
+  wcag21:1.4.12: # Text Spacing (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+    - Jey Nandakumar
+---
+
+## Applicability
+
+This rule applies to any HTML element that is [visible][] and for which the `style` attribute [declares][declared] the [letter-spacing][] CSS property.
+
+## Expectation
+
+For each test target, one of the following is true:
+
+- **not important**: the [computed][] value of its [letter-spacing][] property is not [important][]; or
+- **wide enough**: the [computed][] value of its [letter-spacing][] property is at least 0.12 times the [computed][] value of its [font-size][] property; or
+- **cascade**: the [cascaded][] value of its [letter-spacing][] property is not a value [declared][] in its `style` attribute.
+
+## Assumptions
+
+- There is no mechanism available on the page to adjust [letter-spacing][]. If there is such a mechanism, it is possible to fail this rule while [Success Criterion 1.4.12 Text Spacing][sc1412] is still satisfied.
+
+- This rule assumes that WCAG's meaning for the "Letter spacing style property" is the value of the CSS `letter-spacing` property rather than the actual space between letters. The value of the CSS property is _added_ to whichever spacing already exist (for example, due to kerning). Thus, the actual space between letters can be larger than the value of the `letter-spacing` property. If [Success Criterion 1.4.12 Text Spacing][sc1412] is concerned by the actual space between letters, then this rule may fail (with the `letter-spacing` property being too small) while the Success Criterion is still satisfied (with the actual space being enough).
```

Comment:
> If [Success Criterion 1.4.12 Text Spacing][sc1412] is concerned by the actual space between letters

We might want to reach out for the AG also to clarify this issue

Reply:
When writing this, I had a discussion with @WilcoFiers who was firmly into that interpretation, hence the result. Indeed, it might be worth reaching out to AG:
https://github.com/w3c/wcag/issues/1556

Reply:
Early feedbacks seem to agree with this interpretation 😁